topics/genome-annotation/tutorials/repeatmasker/tutorial.md

```diff
@@ -22,10 +22,11 @@ contributors:
   - r1corre
   - stephanierobin
   - erasmusplus
+  - jkreplak
```

Comment:
You should add yourself in the `CONTRIBUTORS.yaml` file at the root of the repo

Reply:
Done !